src/core/server/logging/layouts/conversions/date.ts

```diff
@@ -66,21 +67,27 @@ function validateTimezone(timezone: string) {
 }
 
 function validate(rawString: string) {
-  for (const matched of rawString.matchAll(timestampRegExp)) {
```

Comment:
that was a surprise, since it works in tests

Reply:
What was the issue outside of the scope of tests?

Reply:
I contacted @elastic/kibana-operations to clarify this. The test and runtime envs should be identical

Reply:
https://github.com/elastic/kibana/pull/57467